### PR TITLE
(PC-13852) feat(accessibility): Change various accessibility labels

### PR DESCRIPTION
**Pull request by aliraiki**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-13852

## Checklist

I have:

- [x] Made sure the title of my PR follows the [convention](1) `($jira) $type($scope): $summary`.
- [x] Made sure my feature is working on the relevant real / virtual devices (native and web).
- [ ] Written **unit tests** native (and web when implementation is different) for my feature.
- [ ] Added a **screenshot** for UI tickets.
- [ ] Attached a **ticket number** or **github dev name** for any added TODO / FIXME.
- [ ] Added new reusable components to **AppComponents** page and communicate to the team on slack.
- [ ] If my PR is a bugfix, I add the link of the "résolution de problème sur le bug" [on Notion](2)
- [x] Made sure translations file is up to date (`yarn translations:extract`)

## Screenshots

| iOS | Android | Mobile - Chrome | Desktop - Chrome | Desktop - Safari |
| --: | ------: | --------------: | ---------------: | ---------------: |
|     |         |                 |                  |                  |

[1]: https://github.com/pass-culture/pass-culture-app-native/blob/master/doc/standards/pr-title.md
[2]: https://www.notion.so/passcultureapp/R-solution-de-probl-mes-sur-les-bugs-5dd6df8f6a754e6887066cf613467d0a

### CODE REVIEW

File: __snapshots__/features/search/pages/__tests__/SearchFilter.native.test.tsx.native-snap [hunk 1]
```diff
@@ -691,7 +691,7 @@ exports[`SearchFilter component should render correctly 1`] = `
         }
       />
       <View
-        accessibilityLabel="Fermer la section"
+        accessibilityLabel="Fermer la section [object Object]"
```
Comment (Lucasbeneston) [inline __snapshots__/features/search/pages/__tests__/SearchFilter.native.test.tsx.native-snap:694]: Il doit y avoir un problème avec le label car il y a `[object Object]`

File: src/libs/accessibilityAndTestId.web.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 export function accessibilityAndTestId(accessibilityLabel?: string, testID?: string) {
   return {
     accessible: true,
-    accessibilityLabel: accessibilityLabel || testID,
+    accessibilityLabel,
```
Comment (Lucasbeneston) [inline src/libs/accessibilityAndTestId.web.ts:4]: 👍

File: src/ui/components/inputs/EmailInput.tsx [hunk 3]
```diff
@@ -20,22 +16,13 @@ interface Props extends TextInputProps {
...
-      {!!label && (
-        <React.Fragment>
-          <LabelContainer>
-            <InputLabel htmlFor={emailInputID}>{label}</InputLabel>
-            {!!isRequiredField && <RequiredLabel />}
-          </LabelContainer>
-          <Spacer.Column numberOfSpaces={2} />
-        </React.Fragment>
-      )}
       <TextInput
```
Comment (Lucasbeneston) [inline src/ui/components/inputs/EmailInput.tsx:26]: Pourquoi tu as supprimé le label ?

Comment (aliraiki) [reply src/ui/components/inputs/EmailInput.tsx:26]: Il est déjà dans `TextInput` (il fallait juste passer `label` en prop)

File: __snapshots__/features/search/pages/__tests__/SearchFilter.native.test.tsx.native-snap (hunk 1 above)
Comment (aliraiki) [reply __snapshots__/features/search/pages/__tests__/SearchFilter.native.test.tsx.native-snap:694]: Ah bien vu, les `title` peuvent être des composants aussi

File: src/ui/components/inputs/EmailInput.tsx (hunk 3 above)
Comment (Lucasbeneston) [reply src/ui/components/inputs/EmailInput.tsx:26]: Tu penses que tu peux ajoute la stories d'InputEmail dans cette PR ou dans une prochaine (dans ce cas, il faudra créer le ticket)

**Review by Lucasbeneston — APPROVED** [review]

**Pull request merged**